paddle/cinn/ir/lowered_func.h

```diff
@@ -30,8 +30,10 @@ class _LoweredFunc_;
  * the function signature of generated code.
  */
 struct Argument {
-  //! Input or output.
-  enum class IO { kInput = 0, kOutput = 1 };
+  //! kInput: arg is input
+  //! kOutput: arg is output
+  //! kUnknown: arg maybe input or output
+  enum class IO { kInput = 0, kOutput = 1, kUnknown = 2 };
```

Comment:
Is IO a good name? Because IO is also related to print\file\hardware\network, can we name it as `InputOutputType`? `ArgInOutType` or something else?

Reply:
Indeed, this name is not very good, but so many modules of the current CINN have relied on this IO. It is too troublesome to modify. This is an enum property in ir::Argument,  the ambiguity doesn't seem too serious.